app/controllers/green_lanes/check_your_answers_controller.rb

```diff
@@ -13,6 +13,8 @@ def show
       @category_one_assessments = determine_category.cat1_with_exemptions
       @category_two_assessments_without_exemptions = determine_category.cat2_without_exemptions
       @category_two_assessments = determine_category.cat2_with_exemptions
+      @resulting_category = prettify_category(resulting_category)
```

Comment:
we need to know the `resulting_category` here
to build the URL `/result?category=x`